src/main/java/server/ChatServer.java

```diff
@@ -8,118 +8,78 @@
 
 // Server class
 public class ChatServer {
-
     // Vector to store active clients
     static Vector<ClientHandler> ar = new Vector<>();
 
+    public static void enqueue(Runnable r) {
```

Comment:
Just abstracting enqueuing tasks to a central location so it could be easily modified to use a thread pool